matrix-meetings-widget/src/components/common/DateTimePickers/CalendarDayPicker.test.tsx

```diff
@@ -80,6 +81,54 @@ describe('<CalendarDayPicker>', () => {
     expect(await axe(container)).toHaveNoViolations();
   });
 
+  it('should have date calendar weekdays starting from Sunday for english locale', async () => {
+    const { container } = render(
```

Comment:
should we not set the locale explicitly to 'en' here?

Reply:
'en' is set globally in the tests and we don't set it explicitly in other similar existing tests, so I just follow existing pattern here, thought it is a bit implicit.